modules/atom/include/Chain.h

```diff
@@ -12,9 +12,47 @@
 #include "Hierarchy.h"
 #include <IMP/macros.h>
 #include <IMP/Decorator.h>
+#include <IMP/base_types.h>
 
 IMPATOM_BEGIN_NAMESPACE
 
+typedef Key<IMP_CHAIN_TYPE_INDEX> ChainType;
+IMP_VALUES(ChainType, ChainTypes);
+
+/** \class IMP::atom::ChainType
+    \brief The type for a chain.
+
+    A given chain is either a Protein, DNA, or RNA polymer.
+
+    The standard chain types are provided with names like
+    IMP::atom::Protein. New chain types can be added simply by creating an
+    instance of ResidueType("my_chain_type").
```

Comment:
Should be ChainType, not ResidueType. Although I would discourage people from creating random new chain types anyway (since RMF won't support them, for example).

Reply:
Sure, sorry, cut-and-paste mistake.